frontend/src/features/permits/hooks/hooks.ts

```diff
@@ -382,16 +382,20 @@ export const useApplicationsInProgressQuery = ({
   page = 0,
   take = 10,
   searchString = "",
-  sorting = [],
+  orderBy = [],
+}: {
+  page?: number;
+  take?: number;
+  searchString?: string;
+  orderBy?: SortingConfig[];
 }) => {
-  const applicationsInProgressQuery = useQuery({
+  return useQuery({
     queryKey: ["applicationInProgress"],
```

Comment:
Because you're not using the query params in the query key, pagination is broken. Did you test your changes completely?
<img width="1078" alt="image" src="https://github.com/bcgov/onroutebc/assets/84348052/167d8f16-3cfc-420f-958a-d44a5014e9f4">

Reply:
thanks - I was missing the `page` parameter in the `queryKey`. Updated.